packages/cms-cli/commands/init.js

```diff
@@ -56,54 +108,43 @@ const action = async options => {
     process.exit(1);
   }
 
-  trackAuthAction(
-    COMMAND_NAME,
-    PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-    TRACKING_STATUS.STARTED
-  );
+  trackAuthAction(COMMAND_NAME, authType, TRACKING_STATUS.STARTED);
 
   createEmptyConfigFile();
   handleExit(deleteEmptyConfigFile);
 
   try {
-    const configData = await personalAccessKeyPrompt({ env });
-    const { name } = await promptUser([PORTAL_NAME]);
-
-    await updateConfigWithPersonalAccessKey(
-      {
-        ...configData,
-        name,
-      },
-      true
-    );
-
+    const { portalId } = await CONFIG_CREATION_FLOWS[authType](env);
     const path = getConfigPath();
-    const portalId = getPortalId();
 
     logger.success(
       `The config file "${path}" was created using your personal access key for portal ${portalId}.`
     );
 
-    trackAuthAction(
-      COMMAND_NAME,
-      PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-      TRACKING_STATUS.COMPLETE,
-      portalId
-    );
+    trackAuthAction(COMMAND_NAME, authType, TRACKING_STATUS.COMPLETE, portalId);
+    process.exit();
```

Comment:
Why is this necessary?

Reply:
Alright. I would feel more comfortable if we knew "why" it was necessary and why we only need the exit in the `try` and not the `catch`.